Form-Controls/index.html

```diff
@@ -13,15 +13,47 @@
     <h1>Product Pick</h1>
   </header>
   <main>
-  <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
+  <form id="orderForm" onsubmit="return validateForm()">
+    <label for="name">Full Name:</label>
+    <input type="text" id="name" name="name">
+    <div id="nameError" class="error"></div>
+
+    <label for="email">Email Address:</label>
+    <input type="email" id="email" name="email">
+    <div id="emailError" class="error"></div>
+
+    <label>Colour:</label>
+    <input type="radio" id="red" name="colour" value="Red">
+    <label for="red">Red</label>
+    <input type="radio" id="blue" name="colour" value="Blue">
+    <label for="blue">Blue</label>
+    <input type="radio" id="black" name="colour" value="Black">
+    <label for="black">Black</label>
+    <div id="colourError" class="error"></div>
+
+    <label for="size">Size:</label>
+    <select id="size" name="size">
+        <option value="">--Select Size--</option>
+        <option value="XS">XS</option>
+        <option value="S">S</option>
+        <option value="M">M</option>
+        <option value="L">L</option>
+        <option value="XL">XL</option>
+        <option value="XXL">XXL</option>
+    </select>
+    <div id="sizeError" class="error"></div>
+
+    <label for="deliveryDate">Delivery Date:</label>
+    <input type="date" id="deliveryDate" name="deliveryDate">
```

Comment:
Is this a constrained time window? The AC says:
 I must collect a date and make sure that date is in the next four weeks.
 
 How could this be achieved only using html and css?